scripts/deploy/publish-build-artifacts.sh

```diff
@@ -95,7 +95,8 @@ publishPackage() {
   # Replace the version in every file recursively with a more specific version that also includes
   # the SHA of the current build job. Normally this "sed" call would just replace the version
   # placeholder, but the version placeholders have been replaced by "npm_package" already.
-  sed -i "s/${buildVersion}/${buildVersionName}/g" $(find . -type f -not -path '*\/.*')
+  escapedVersion=$(echo ${buildVersion} | sed 's/[.[\*^$]/\\&/g')
```

Comment:
not a big fan of escaping like that, but seems like the most reasonable solution. Simple `sed` to prepend a backslash for escaping.

Reply:
Yeah, its definitely not super readable, but I don't have a better option myself.